README.md

```diff
@@ -1,11 +1,11 @@
 # diffmin
 
-In a software development process based on code reviews, the visualization of 
-the diff between the base, and the target branch is of utmost importance. It is
-known that spurious formatting changes pollute code reviews and lead to
-unnecessary communication and code review work. Today, the only solution for
-minimizing the diff is one that ignores white spaces. This project will aim to
-automatically minimize pull requests by keeping only the AST changes.
+This application patches a file using the diff generated by
+[gumtree-spoon-ast-diff](https://github.com/SpoonLabs/gumtree-spoon-ast-diff).
+It is similar to the
+[patch utility](https://man7.org/linux/man-pages/man1/patch.1.html) in Linux
+except diffmin also aims to minimise the resultant diff between the two files
```

Comment:
@slarse , I remember that you were suggesting to make `diffmin` only for applying patches and we would take care of pretty-printing in some other project. Let me know what you think of this and I will rewrite it if needed.

Reply:
Well, I was suggesting making the patch application engine a standalone thing, or possibly incorporate straight into `gumtree-spoon`, because that's something that have been requested a few times in the past. The goal of diffmin itself is and always will be to minimize textual diffs in patches :)

I think the description is good. We can elaborate on this once diffmin is a bit more complete. I also think that it's pretty hard to understand the concept without a concrete example, but I feel like the best way to incorporate an example is with actually applying diffmin on something and showing the textual diff before and after (so we can wait with that until we have a complete prototype).

Reply:
Ah, right! Diffmin is supposed to do the pretty-printing stuff as its name suggests.

Yes, we can update the description again when we have a concrete example.